ros2/rosdistro-testing
# sros2: 0.5.0-0 in 'bouncy/distribution.yaml' [bloom]

Increasing version of package(s) in repository `sros2` to `0.5.0-0`:

- upstream repository: https://github.com/ros2/sros2.git
- release repository: https://github.com/ros2-gbp/sros2-release.git
- distro file: `bouncy/distribution.yaml`
- bloom version: `0.6.5`
- previous version for package: `null`

## sros2

```
* Update docs for bouncy leveraging remapping for demo (#53 <https://github.com/ros2/sros2/issues/53>)
* Windows tutorial tweaks (#58 <https://github.com/ros2/sros2/issues/58>)
* publish ans subscribe to all parameter service topics (#52 <https://github.com/ros2/sros2/issues/52>)
* remove partitions (#45 <https://github.com/ros2/sros2/issues/45>)
* as of Bouncy access control is available for both Fast-RTPS and Connext (#50 <https://github.com/ros2/sros2/issues/50>)
* add pytest markers to linter tests
* Remove outdated docker resources now that SROS2 ships as part of the core (#48 <https://github.com/ros2/sros2/issues/48>)
* add X509 extensionCA:false (#47 <https://github.com/ros2/sros2/issues/47>)
* enable_liveliness_protection (#44 <https://github.com/ros2/sros2/issues/44>)
* set zip_safe to avoid warning during installation (#42 <https://github.com/ros2/sros2/issues/42>)
* Linter fixup
* add special service rule only if not wildcarding everything (#40 <https://github.com/ros2/sros2/issues/40>)
* remove whant now appears to be obsolete DCPS whitelisting (#34 <https://github.com/ros2/sros2/issues/34>)
* fix sample_policy download command (#37 <https://github.com/ros2/sros2/issues/37>)
* Fix access control for ardent (#33 <https://github.com/ros2/sros2/issues/33>)
* advise to ask questions on ROS answers
* print full help when no command is passed (#35 <https://github.com/ros2/sros2/issues/35>)
* add return code to all verb apis (#28 <https://github.com/ros2/sros2/issues/28>)
* Contributors: Dirk Thomas, Mikael Arguedas, Shane Loretz, dhood
```
